styles.css

```diff
@@ -1,3 +1,793 @@
+    margin: auto;
+}
+.input-group {
+    display: flex;
+}
+.leave-reply {
+    text-align: left;
+    padding: 0 646px 0 254px;
+}
+.input-reply-group {
+    display: flex;
+}
+input[class~="name"] {
+    width: 240px;
+    margin-right: 20px;
+}
+input[class~="email"] {
+    width: 430px;
+}
+input[class~="reply-name"],
+input[class~="reply-phone"] {
+    width: 325px;
+    margin-right: 20px;
+
+}
+input[class~="reply-email"],
+input[class~="reply-website"] {
+    width: 655px;
+
+}
+.details-reply-textarea{
+    resize: none;
+    height: 150px;
+    width: 972px;
+}
+.details-textarea  {
+    resize: none;
+    height: 150px;
+}
+
+.email-input::placeholder, 
+.details-input::placeholder,
+.details-textarea::placeholder,
+.details-reply-textarea::placeholder {
+    color: #C2C3C5;
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 26px;
+    text-align: left;
+
+}
+
+.email-btn {
+    background-color: #21E786;
+    padding: 12px 24px;
+    width: 154px;
+    height: 56px;
+    border: none;
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #040B11;
+}
+
+.search-bar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 280px;
+    height: 48px;
+    background-color: #141B22;
+    border: none;
+}
+
+.search-input {
+    border: none;
+    width: 280px;
+    height: 48px;
+    background-color: transparent;
+    padding-left: 20px;
+    color: #fff
+}
+
+.search-input::placeholder {
+    color: #888B8E;
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 26px;
+}
+
+.search-btn {
+    background-color: transparent;
+    appearance: none;
+    border: none;
+    padding-right: 20px;
+    cursor: pointer;
+}
+
+.aside-filter {
+    display: flex;
+    flex-direction: column;
+    width: 280px;
+    height: 970px;
+
+}
+.aside-form {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: 12px;
+}
+.filter-box {
+    display: flex;
+    flex-direction: column;
+    width: 280px;
+    border: none;
+    background-color: #141B22;
+}
+
+.details-box {
+    display: flex;
+    width: 280px;
+    height: 234px;
+}
+
+.filter-title {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    text-align: left;
+    padding: 16px 12px;
+    list-style: none;
+}
+.filter-box[open] .filter-title {
+    border-bottom: 1px solid #434343;
+}
+.filter-box[open] {
+    content: '';
+    display: flex;
+    background-image: url("data:image/svg+xml,%3Csvg width='16' height='4' viewBox='0 0 16 4' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M14.5 2L8.00004 2L1.5 2' stroke='%2321E786' stroke-width='3' stroke-miterlimit='10' stroke-linecap='square' stroke-linejoin='round'/%3E%3C/svg%3E%0A");
```

Comment:
I’m glad to see that you found the method of embedding images with Base64 useful.